### PR TITLE
move md5 function to lib dir and symlink to fastqc test dir, rename m…

### PR DESCRIPTION
**Pull request by stevekm**
…d5 function

### CODE REVIEW

**Comment by ewels** [comment]
CI tests are royally messed up. Fixed in https://github.com/nf-core/modules/pull/49

Hopefully if this is merged and you can pull in the changes, the FastQC tests should fire on this PR properly... 🤞

**Comment by ewels** [comment]
Ok, I updated against the latest master - test should run successfully not. However I disabled the file hash check, maybe we can play with it and get it enabled again?

**Comment by ewels** [comment]
Is scrollspy not already in place somewhere? eg. the sidebar at https://nf-co.re/sarek/docs/usage

**Comment by stevekm** [comment]
@ewels  was there anything else that needed to be done for this?

**Comment by ewels** [comment]
> Is scrollspy not already in place somewhere? eg. the sidebar at https://nf-co.re/sarek/docs/usage

How did this comment get here? That was meant to be on a different PR 😅 

I'm happy - just need to fix the failing code style CI test.

Generally once you have an approved review and all tests are passing we say that you can self-merge your own PRs.

File: lib/checksum.groovy [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    //This bytes[] has bytes in decimal format;
+    //Convert it to hexadecimal format
+    StringBuilder sb = new StringBuilder();
+    for(int i=0; i< bytes.length ;i++)
+    {
+        sb.append(Integer.toString((bytes[i] & 0xff) + 0x100, 16).substring(1));
+    }
+
+    //return complete hash
+   return sb.toString();
```
Comment (ewels) [inline lib/checksum.groovy:33]:
```suggestion
    return sb.toString();
```

**Pull request merged**